pluginconfig/pluginconfig-311.yaml

```diff
@@ -301,7 +301,7 @@ versions:
       startup: quay.io/openshift-on-azure/ci-azure:latest
       sync: quay.io/openshift-on-azure/ci-azure:latest
       tlsProxy: quay.io/openshift-on-azure/ci-azure:latest
-      logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod10182019
+      logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod11012019
```

Comment:
Hi, @olga-mir , in addition to  agent image version change and there are other changes such as

- change from "deployments" to "replicasets"
- change the dockerProviderVersion from "7.0.0-5"  to "7.0.0-6"
- other changes as well

Can you please look at this PR  https://github.com/microsoft/OMS-docker/pull/335/files and make the applicable updates  to this PR.

Reply:
Hi @ganga1980 thanks for reviewing. I've updated PR to include these changes.

Reply:
@olga-mir, changes looks good. I see the cluster in the ux and all the data getting ingested.

Reply:
lgtm